src/partielspy/group.py

```diff
@@ -45,21 +46,30 @@ def __deepcopy__(self, memo):
         }
         return res
 
-    def _to_xml(self, node: etree):
+    def _from_xml(self, node: etree.Element):
+        self.name = node.get("name", self.name)
+        track_layouts = node.findall("layout")
+        for track_layout in track_layouts:
+            track_layout_value = track_layout.get("value")
+            track_node = node.getparent().find(
```

Comment:
Another solution would be to accept a `list[etree.Element]` as the argument with the group and the tracks. What do you think?